tests/genai/scorers/test_builtin_scorers.py

```diff
@@ -354,8 +359,12 @@ def test_relevance_to_query(mock_is_context_relevant):
     )
 
 
-@databricks_only
-def test_safety():
+def test_safety(is_in_databricks):
+    if not is_databricks_uri(mlflow.get_tracking_uri()):
```

Comment:
```suggestion
    if not is_in_databricks:
```

can we use `is_in_databricks`?

Reply:
Got it. Can we isolate this test?

```python
def test_safety_non_databricks():
    mlflow.set_tracking_uri("mlruns")
    with pytest.raises(MlflowException, match=r"The Safety scorer is only available"):
        Safety()
```